oddsgym/envs/soccer.py

```diff
@@ -37,9 +37,9 @@ def render(self, mode='human'):  # pragma: no cover
         msg : str
             A string with the current team names, balance and step.
         """
-        return 'Home Team {} VS Away Team {}. {}'.format(self.teams[self.current_step % self._odds.shape[0]][0],
+        print('Home Team {} VS Away Team {}. {}'.format(self.teams[self.current_step % self._odds.shape[0]][0],
                                                          self.teams[self.current_step % self._odds.shape[0]][1],
-                                                         super().render(mode))
+                                                         super().render(mode)))
```

Comment:
Does this work? After the change, render returns None, so it won't add the current step & balance when rendering.